examples/get-new-releases/backend_server/Ballerina.toml

```
@@ -0,0 +1,13 @@
+[package]
+org = "spotify_backend"
```

Use the org as WSO2 in examples.
```suggestion
org = "wso2"
```